website/docs/tutorials/07-testing/02-cheatcodes/given.md

```diff
@@ -31,3 +31,6 @@ func test_less_equal_compare{syscall_ptr : felt*, range_check_ptr}(a : felt, b :
     return ();
 }
 ```
+
+Note: There is also the [`example`](./example.md) cheatcode that tests certain cases.
+This may come handy if there is a need to test specific edge cases.
```

Comment:
```suggestion
:::info
There is also the [`example`](./example.md) cheatcode that tests explicitly provided cases. It may come handy if you don't want to rely on randomization.
:::
```

Reply:
[update docs](https://github.com/software-mansion/protostar/pull/947/commits/75b02de8f52d7eced8f64ae8687db4947c59bb45)